app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_toolbar.less

```diff
@@ -0,0 +1,89 @@
+.toolbar {
+    .modes {
+        margin-right: 7px;
+    }
+    .modes-mode {
+        .lib-vendor-prefix-display(flex);
+        align-items: center;
+        justify-content: center;
+        width: 32px;
+        height: 32px;
+        padding: 0;
+        &:not(:last-child) {
+            margin-right: 5px;
+        }
+        text-decoration: none;
+        border: none;
+        background-color: @color-maksym__main-light-gray;
+        transition: background-color 0.4s ease 0s;
+        &::before {
+            display: contents;
+            width: 14px;
+            height: 14px;
+            color: @color-maksym__main-gray;
+            transition: color 0.4s ease 0s;
+        }
+        &:hover,
+        &:focus {
+            background-color: @color-maksym__main-black;
+            &::before {
+                color: @color-maksym__white;
+            }
+        }
+        &.active {
+            &::before {
+                color: @color-maksym__white;
+            }
+            background-color: @color-maksym__main-black;
+        }
+    }
+    .mode-grid,
+    .mode-list {
+    }
+
+    &-products {
+    }
+    &-amount {
+        float: right;
+        padding: 10px 0 10px 0;
+        text-transform: uppercase;
+        border-bottom: 1px solid @color-maksym__main-gray;
+    }
+    &-sorter {
+        float: left;
+    }
+    .sorter {
+        .lib-vendor-prefix-display(flex);
+        align-items: center;
+        width: min-content;
+        padding: 0;
+        &-label {
+            .lib-visibility-hidden();
+            width: 0;
+        }
+        .select2 {
+            &-container {
+            }
```

Comment:
Don't leave code like this to decrease CSS file size

Reply:
done